packages/office-ui-fabric-react/src/components/Dialog/Dialog.tsx

```diff
@@ -63,7 +63,7 @@ export class Dialog extends BaseComponent<IDialogProps, IDialogState> {
     let subTextContent;
     const dialogClassName = css(this.props.className, {
       ['ms-Dialog--lgHeader ' + styles.isLargeHeader]: type === DialogType.largeHeader,
-      ['ms-Dialog--close ' + styles.isClose]: type === DialogType.close,
+      ['ms-Dialog--close ' + styles.isClose]: type === DialogType.close
```

Comment:
Nit: No reason to remove the comma. Having the comma makes merges/diffs easier when people add new lines to the object.